pkg/oauth/registry/helpers/strategy.go

```diff
@@ -0,0 +1,19 @@
+package helpers
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util/validation/field"
+)
+
+var CannotCreateStrategy = readOnly{kapi.Scheme}
```

Comment:
I planned on reusing it for other OAuth tokens since they will follow the same pattern of only allowing reads and deletes.